docs/root/configuration/http/http_filters/router_filter.rst

```diff
@@ -374,6 +374,19 @@ compared to network latency between client and Envoy. This header is set on resp
 
 .. _config_http_filters_router_x-envoy-overloaded_set:
 
+x-envoy-upstream-stream-duration-ms
```

Comment:
Also can you add one liner and link to this from docs/root/version_history/current.rst?

Reply:
done